cypress/e2e/po/other-products/cis-benchmark-list.po.ts

```diff
@@ -0,0 +1,19 @@
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+
+export default class CisBenchmarkListPo extends BaseResourceList {
+  clickCreate() {
+    return cy.get('[data-testid="masthead-create"]').click();
```

Comment:
This button is in the List page, it isn't part of the Resource list and is at the dashboard-content level